internal/core/metadata/application/device.go

```diff
@@ -63,6 +67,16 @@ func AddDevice(d models.Device, ctx context.Context, dic *di.Container) (id stri
 		correlation.FromContext(ctx),
 	)
 
+	// If device is successfully created, check each AutoEvent interval value and display a warning if it's smaller than the suggested 10ms value
+	for _, autoEvent := range d.AutoEvents {
+		valid, err := utils.CheckDuration(autoEvent.Interval, minAutoEventInterval)
+		if err != nil {
+			lc.Warnf("failed to check auto event interval: %v", err)
```

Comment:
if there is anything parsing wrong, it should log as error.
this part should be moved into the checking function.